metrics/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
+
+#[macro_use]
+extern crate derive_builder;
```

Comment:
This syntax is a [bit legacy](https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#macros), and you should probably just import the macro you're using in modules 

```rust
use derive_builder::Builder;

#[derive(Debug, Builder)]
#[builder(build_fn(validate = "Self::validate"))]
pub struct Config {}
```